src/linux/py_proc.h

```diff
@@ -127,10 +125,9 @@ _py_proc__analyze_elf64(py_proc_t * self) {
     ) {
       Elf64_Sym * sym      = (Elf64_Sym *) (elf_map + tab_off);
       char      * sym_name = (char *) (elf_map + p_strtabsh->sh_offset + sym->st_name);
-      // ASLR: This leads to some good corrections, but it still fails in some cases.
-      void      * value    = (void *) sym->st_value >= self->map.bss.base && (void *) sym->st_value < self->map.bss.base + self->map.bss.size \
+      void      * value    = ehdr.e_type == ET_EXEC \
```

Comment:
Assuming that `base` is now `self->map.elf.base`, I would say this should simply be
`value = base + sym->st_value`
and the ternary operator is not required anymore.